docs/API.md

```diff
@@ -7,47 +7,28 @@ hidden: true
 slug: ue-api
 ---
 
-## Table of content
-
-- [Init SDK](#init-sdk)
-- [Start](#start)
-- [Log Event](#log-event)
-- [Get AppsFlyerUID](#get-appsflyeruid)
-- [Set Custom User Id](#set-custom-user-id)
-- [Uninstall tracking](#uninstall)
-  - [iOS Uninstall](#ios-uninstall)
-  - [Android Uninstall](#android-uninstall)
-- [Set Additional Data](#set-additional-data)
-
 ## Init SDK
 
 <img src="https://files.readme.io/70a3db6-ProjectSettings2.png"  width="650"/>
 
-- **Dev key** - Found under App Settings in your AppsFlyer Dashboard.
-- **App id** - app id from the itunes store (without the 'id' prefix)
-- **Is debug** - Used to dub to AppsFlyer SDK. (Development Only!). Once this is set up the AppsFlyer SDK can log all Installs and Sessions.
-
-- **Currency Code (only iOS)** - Defines the app currency method
-
-- **Disable SKAdNetwork(Only iOS)** - Disable SKAdNetwork sessions
-
-- **Automatically start the AppsFlyer SDK** - When set to true, the SDK will be sending a session automatically (before Blueprint events). If set to false, it's the developer responsability to call the start API from the Blueprint.
+- **AppsFlyer Dev Key** - Obtain the Dev Key from the AppsFlyer Dashboard in the App Settings section.
+- **Apple App ID** - Obtain the app ID from the iTunes store (without the `id` prefix)
+- **Is debug** - Once this is enabled the AppsFlyer SDK can log all Installs and Sessions.
+- **Currency Code (only iOS)** - Set the app currency.
+- **Disable SKAdNetwork (Only iOS)** - Disable SKAdNetwork sessions.
+- **Automatically start the AppsFlyer SDK** - When set to `true`, the SDK sends the session automatically (before Blueprint events). When set to `false`, it's the developer's responsibility to call the start API from the Blueprint.
 
 ## Start
 
 - Starts the SDK by sending the session to the server.
+- By default, the `start` method is called automatically once the app is being launched.
+- If needed, this setting can be disabled in the Plugin settings, and the developer can call the `start` method under the blueprint. When doing so, the SDK will send a session on every background-foreground transition.
 
-- As a default, the start method is being called automatically once the app is being launch.
-
-- If needed, this setting can be disabled in the Plugin settings, and the developer can call the start method under the blueprint. When doing so, the SDK will send a session on every background-foreground transition.
-<img src="https://files.readme.io/6fe8c83-autoStartOff.png"  width="400"/>

```

Comment:
Put back all the HTML style image references. They allow to determine the width, unlike markdown